OOP_PHONE.cpp

```diff
@@ -0,0 +1,277 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <sstream>
+#include <vector>
+using namespace std;
+
+class Person
+{
+public:
+    string name;
+    string surname;
+    string separator;
+    string phoneNumber;
+};
+
+class Helper
+{
+public:
+    Person **sortName(Person **arr, string sign, string elem, int count)
+    {
+        for (int i = 0; i < count; i++)
+        {
+            for (int j = 0; j < count - 1 - i; j++)
+            {
+                bool tmp;
+                if (sign == "Ascending")
+                {
+                    if (elem == "Surname")
+                    {
+                        tmp = arr[j]->surname[0] > arr[j + 1]->surname[0];
+                    }
+                    else
+                    {
+                        tmp = arr[j]->name[0] > arr[j + 1]->name[0];
+                    }
+                }
+                else if (sign == "Descending")
+                {
+                    if (elem == "Surname")
+                    {
+                        tmp = arr[j]->surname[0] < arr[j + 1]->surname[0];
+                    }
+                    else
+                    {
+                        tmp = arr[j]->name[0] < arr[j + 1]->name[0];
+                    }
+                }
+                else
+                {
+                    cout << "Error";
+                }
+                if (tmp)
+                {
+                    Person *x = arr[j + 1];
+                    arr[j + 1] = arr[j];
+                    arr[j] = x;
+                }
+            }
+        }
+        return arr;
+    }
+
+    Person **sortPhoneNumber(Person **arr, string sign, int count)
+    {
+        for (int i = 0; i < count; i++)
+        {
+            for (int j = 0; j < count - 1 - i; j++)
+            {
+                bool tmp;
+                if (sign == "Ascending")
+                {
+                    tmp = arr[j]->phoneNumber[0] > arr[j + 1]->phoneNumber[0] ||
+                          arr[j]->phoneNumber[1] > arr[j + 1]->phoneNumber[1] ||
+                          arr[j]->phoneNumber[2] > arr[j + 1]->phoneNumber[2];
+                }
+                else if (sign == "Descending")
+                {
+                    tmp = arr[j]->phoneNumber[0] < arr[j + 1]->phoneNumber[0] ||
+                          arr[j]->phoneNumber[1] < arr[j + 1]->phoneNumber[1] ||
+                          arr[j]->phoneNumber[2] < arr[j + 1]->phoneNumber[2];
+                }
+                if (tmp)
+                {
+                    Person *x = arr[j + 1];
+                    arr[j + 1] = arr[j];
+                    arr[j] = x;
+                }
+            }
+        }
+        return arr;
+    }
+
+    void sort(Person **arr, int count)
+    {
+        string x;
+        string y;
+        cout << "Please choose an ordering to sort: 'Ascending' or 'Descending'." << endl;
+        cin >> x;
+        if (x == "Ascending" || x == "Descending")
```

Comment:
better to check if its not `Ascending` or `Descending` constants to throw error first.